### ISSUE TITLE
Table: Not keyboard accessible

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-js#52]
_From @mikewheaton on June 5, 2016 19:42_

For tables with selectable rows, it should be possible to arrow between the rows and use enter/space to select a row. Refer to OneDrive's file list as a reference.

![image](https://cloud.githubusercontent.com/assets/1382445/15807771/fca2b514-2b1a-11e6-9981-4c0b62f34423.png)

_Copied from original issue: OfficeDev/office-ui-fabric-core#633_

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
